File: 215 (Kth Largest Element in an Array)/python/Solution.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:

        mheap = []

        for i in range(k):
            heapq.heappush(mheap, nums[i])
        

        for i in range(k,len(nums)):
            if nums[i]>mheap[0]:
                heapq.heappop(mheap)
                heapq.heappush(mheap,nums[i])

        return mheap[0]

File: 215 (Kth Largest Element in an Array)/javascript/Solution.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var findKthLargest = function(nums, k) {
  const mheap = new MinPriorityQueue();

  for(let i = 0;i<k;i++){
      mheap.enqueue(nums[i]);
  }

  for(let i = k;i<nums.length;i++){
      if (nums[i]>mheap.front().element){
          mheap.dequeue();
          mheap.enqueue(nums[i]);
      }
  }
  return mheap.front().element;
};